example/.kube.yml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+
+metadata:
+  name: {{.app}}-{{.env}}
+
+spec:
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: {{.app}}
+        env: {{.env}}
+    spec:
+      containers:
+        - name: app
+          image: {{.image}}
+          ports:
+            - containerPort: 8080
+---
+apiVersion: v1
+kind: Service
```

Comment:
Are we updating this according to our recommendations? (i.e. Ingress)

Reply:
No, I want a simple and runnable example across all platforms.

Reply:
Gotcha, thanks!